lib/components/WithdrawForm.jsx

```diff
@@ -59,6 +60,14 @@ const handleWithdrawInstantly = async (
   )
 }
 
+const parseNumString = (amount, decimals) => {
+  try {
```

Comment:
This prevents the app from crashing if they enter in too many decimal digits, etc

Reply:
http://localhost:3000/pools/rinkeby/0x95b81683E9e5e611c37FF67236a640ECf72f5082/home crashed for me trying to deposit `60.8669333900000000000000000000000000000000`, this should probably be a util function

Reply:
It also probably makes more sense to chop the string at the max decimals rather than returning 0

Reply:
`60.8669333900000000000000000000000000000000` that's ... excessive. 🙈

Reply:
Ah I see, I forgot to extract that method and use it elsewhere. Done! 👍